Pulsarc/Utils/KeyVal.cs

```diff
@@ -1,20 +1,15 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace Pulsarc.Utils
+﻿namespace Pulsarc.Utils
 {
     public class KeyVal<Key_, Value_>
     {
         public Key_ Key { get; set; }
         public Value_ Value { get; set; }
-
         public KeyVal() { }
 
         public KeyVal(Key_ key, Value_ val)
         {
-            Key = key;
-            Value = val;
+            this.Key = key;
```

Comment:
Why add unnecessary this ?

Reply:
To use those values in that instance, those are indeed unnecessary if you plan not adding anything else in that class that uses those variables.

Reply:
This object is just a workaround since KeyValuePair is read-only, and not this one

Reply:
Changed it back to how it was before.